boards/nxp/mimxrt1050_evk/revision.cmake

```diff
@@ -0,0 +1,7 @@
+if (NOT DEFINED BOARD_REVISION)
+	set(BOARD_REVISION "hyperflash")
+else ()
```

Comment:
this should not be needed

Reply:
why not? its the way I got it to work, how else do you do it?